core/src/main/java/org/geysermc/geyser/text/MinecraftLocale.java

```diff
@@ -169,23 +198,21 @@ private static boolean loadLocale(String locale) {
                 langMap.put(entry.getKey(), entry.getValue().asText());
             }
 
-            String bedrockLocale = locale.toLowerCase(Locale.ROOT);
-            if (bedrockLocale.equals("no_no")) {
-                // Store this locale under the Bedrock locale so we don't need to do this check over and over
-                bedrockLocale = "nb_no";
-            }
-
-            // Insert the locale into the mappings
-            LOCALE_MAPPINGS.put(bedrockLocale, langMap);
+            return langMap;
+        } catch (IOException e) {
+            throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.file", locale, e.getMessage()));
+        }
+    }
 
-            try {
-                localeStream.close();
-            } catch (IOException e) {
-                throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.file", locale, e.getMessage()));
-            }
-            return true;
-        } else {
-            return false;
+    /**
+     * Add more translations to a locale if it's loaded.
+     *
+     * @param locale Locale to load
+     * @param langMap Map of new translations to add
+     */
+    public static void addCustomTranslations(String locale, Map<String,String> langMap){
```

Comment:
formatting, please add a space in between the closing bracket and curly bracket, as shown below -
```suggestion
    public static void addCustomTranslations(String locale, Map<String,String> langMap) {
```

Reply:
same with isLocalLoaded (shouldnt that be `isLocaleLoaded`?) below :)